README.md

```diff
@@ -3357,6 +3203,46 @@ not be solely relied upon.
 > resolved in the latest versions, it is important to note that bad hints can
 > still cause slowdowns by guiding the solver in the wrong direction.
 
+Often, you may need to explore the impact of forcing certain variables to
+specific values. To avoid copying the entire model multiple times to set the
+values of variables explicitly, you can also use hints to fix variables their
+hinted value with the following parameter:
+
+```python
+solver.parameters.fix_variables_to_their_hinted_value = True
```

Comment:
If I remember correctly, `fix_variables_to_their_hinted_value` does not work the same way as assumptions and will only be applied after presolve. If the model is solved in presolve, the solution will not match the hints.

Reply:
https://github.com/google/or-tools/commit/dd268623a6835100f8cf5b1a8c548b0d7b025562 this seems to be the commit I think

Reply:
Maybe I will add a comment on that later. I have not too much confidence that this will work reliably and it should probably not be used in production code but only for testing purposes.